iOS/Features/HomeMapScene/Sources/HomeMapScene/HomeMapButtonView.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  HomeMapButtonView.swift
+//  Test
+//
+//  Created by 윤동주 on 11/22/23.
+//
+
+import UIKit
+
+enum ButtonImage: String {
+    case setting = "gearshape.fill",
+         map = "map.fill",
+         location = "mappin"
+}
+
+/// HomeMap 내의 버튼들을 감싸는 View
+final class HomeMapButtonView: UIView {
+
+    // MARK: - Properties
```

Comment:
```suggestion
    // MARK: - UI Components
```

https://www.notion.so/musicspot/Swift-c80e89fdf4ee41209c7afaed3d759e82?pvs=4#2ca9bc36ba1640939eb82712e2db8b65

조금 헷갈리는 면이 있었던 것 같아서
MARK 컨벤션에 대한 예시를 추가해뒀습니다..!
한 번 확인 부탁드려요 😉